src/advertising_categories.h

```diff
@@ -0,0 +1,211 @@
+	advertising_categories["science-geography"] = 141;
+	advertising_categories["science-geology"] = 142;
+	advertising_categories["science-mathematics"] = 143;
+	advertising_categories["science-mechanics"] = 144;
+	advertising_categories["science-palaeontology"] = 145;
+	advertising_categories["science-physics"] = 146;
+	advertising_categories["science-science"] = 147;
+	advertising_categories["society-anthropology"] = 148;
+	advertising_categories["society-dating"] = 149;
+	advertising_categories["society-gay life"] = 150;
+	advertising_categories["society-marriage"] = 151;
+	advertising_categories["society-social networking"] = 152;
+	advertising_categories["society-society"] = 153;
+	advertising_categories["society-sociology"] = 154;
+	advertising_categories["sports-american football"] = 155;
+	advertising_categories["sports-archery"] = 156;
+	advertising_categories["sports-athletics"] = 157;
+	advertising_categories["sports-baseball"] = 158;
+	advertising_categories["sports-basketball"] = 159;
+	advertising_categories["sports-bodybuilding"] = 160;
+	advertising_categories["sports-bowling"] = 161;
+	advertising_categories["sports-boxing"] = 162;
+	advertising_categories["sports-climbing"] = 163;
+	advertising_categories["sports-cricket"] = 164;
+	advertising_categories["sports-cycling"] = 165;
+	advertising_categories["sports-diving"] = 166;
+	advertising_categories["sports-fishing"] = 167;
+	advertising_categories["sports-golf"] = 168;
+	advertising_categories["sports-gymnastics"] = 169;
+	advertising_categories["sports-hockey"] = 170;
+	advertising_categories["sports-hunting"] = 171;
+	advertising_categories["sports-jogging"] = 172;
+	advertising_categories["sports-martial arts"] = 173;
+	advertising_categories["sports-olympics"] = 174;
+	advertising_categories["sports-racing"] = 175;
+	advertising_categories["sports-rugby"] = 176;
+	advertising_categories["sports-sailing"] = 177;
+	advertising_categories["sports-skateboarding"] = 178;
+	advertising_categories["sports-skiing"] = 179;
+	advertising_categories["sports-snowboarding"] = 180;
+	advertising_categories["sports-soccer"] = 181;
+	advertising_categories["sports-sports"] = 182;
+	advertising_categories["sports-surfing"] = 183;
+	advertising_categories["sports-swimming"] = 184;
+	advertising_categories["sports-tennis"] = 185;
+	advertising_categories["sports-volleyball"] = 186;
+	advertising_categories["sports-wrestling"] = 187;
+	advertising_categories["sports-yoga"] = 188;
+	advertising_categories["technology & computing-apple"] = 189;
+	advertising_categories["technology & computing-database"] = 190;
+	advertising_categories["technology & computing-freeware"] = 191;
+	advertising_categories["technology & computing-gaming"] = 192;
+	advertising_categories["technology & computing-programming"] = 193;
+	advertising_categories["technology & computing-software"] = 194;
+	advertising_categories["technology & computing-technology & computing"] = 195;
```

Comment:
Are we intentionally missing macOS and others? Thanks

Reply:
Those were the exact 203 categories specified in the previous model under: 
`bat-native-ads/resources/user_models/languages/en/user_model.json`. This one was made to ensure there was a temporary parity between the two.